proto/rill/admin/v1/api.proto

```diff
@@ -690,6 +690,31 @@ service AdminService {
   rpc ListPublicBillingPlans(ListPublicBillingPlansRequest) returns (ListPublicBillingPlansResponse) {
     option (google.api.http) = {get: "/v1/billing/plans"};
   }
+
+  rpc RequestProjectAccess(RequestProjectAccessRequest) returns (RequestProjectAccessResponse) {
+    option (google.api.http) = {
+      post: "/v1/organizations/{organization}/projects/{project}/access-request",
+      body: "*"
+    };
+  }
+
+  rpc GetProjectAccess(GetProjectAccessRequest) returns (GetProjectAccessResponse) {
+    option (google.api.http) = {get: "/v1/organizations/{organization}/projects/{project}/access-request/{id}"};
+  }
+
+  rpc ApproveProjectAccess(ApproveProjectAccessRequest) returns (ApproveProjectAccessResponse) {
+    option (google.api.http) = {
+      post: "/v1/organizations/{organization}/projects/{project}/access-request/{id}/approve",
+      body: "*"
+    };
+  }
+
+  rpc DenyProjectAccess(DenyProjectAccessRequest) returns (DenyProjectAccessResponse) {
+    option (google.api.http) = {
+      post: "/v1/organizations/{organization}/projects/{project}/access-request/{id}/deny",
+      body: "*"
+    };
+  }
```

Comment:
I would suggest just using the ID of the project access request here (without passing org/project names), since it already has a foreign key to the project ID. This also avoids nullifying all access requests if an org/project is renamed.

Reply:
Got it. Updated